sprocs/assessment_instances_select_log.sql

```diff
@@ -340,7 +340,13 @@ BEGIN
                 WHERE
                     pvl.assessment_instance_id = ai_id
                     AND pvl.page_type = 'studentInstanceQuestion'
-                    AND pvl.authn_user_id = ai.user_id
+                    AND (pvl.authn_user_id = ai.user_id
+                         OR EXISTS (SELECT 1
```

Comment:
It's entirely possible that the query planner/optimizer will do the right thing here by default, but I wonder if this should instead be `OR (ai.group_id IS NOT NULL AND EXISTS (SELECT 1 ... )`, at least until we can get an index on the `group_id` column? An `EXPLAIN ...` could probably help us decide one way or the other with more confidence.

Reply:
That's a good point. I'm not sure the query planner would do the right thing for non-group assessments with views by instructors in this case. I'll add the qualifier.